.gitattributes

```diff
@@ -26,4 +26,4 @@ utils/spdx/src/main/resources/licenserefs/*		linguist-detectable=false
 utils/spdx/src/main/resources/licenses/*		linguist-detectable=false
 
 # Use Unix line endings for the expected control file of a Debian package.
-utils/common/src/test/assets/control-expected.txt	text eol=lf
+utils/common/src/funTest/assets/control-expected.txt	text eol=lf
```

Comment:
I believe that creating a couple of files should not be a reason for turning a test into a funTest.
Maybe only if it is very slow. But IIRC we have so many (non-fun) tests operating on files, that 
moving these three to me seems a bit arbitrary. For example `DirectoryStashTest` probably isn't really slow I blind guess?.

Reply:
I agree that we don't have strict rules here. However, non-functional tests actually should be unit test in the common sense, which usually have IO stuff mocked.

As I didn't want to spend the effort to measure test run times, I simply went with splitting IO and non-IO tests here (not implying that this is what we should generally do), as certainly the archive tests are quite slow, and not putting other IO tests to funTests would also have been quite arbitrary then.

Reply:
> I agree that we don't have strict rules here. However, non-functional tests actually should be unit test in the common sense, which usually have IO stuff mocked.

I prefer to make at least some agreement with the core devs before moving things around, which are not
strictly nececssary in this PR. So, could we limit the move to the deletion tests?

Reply:
Can we do the opposite to avoid another test cycle and merge now, and undo later if the core devs decide so?